mesonbuild/interpreterbase/baseobjects.py

```diff
@@ -82,7 +82,7 @@ def __repr__(self) -> str:
         return f'<[{type(self).__name__}] holds [{type(self.held_object).__name__}]: {self.held_object!r}>'
 
 class RangeHolder(MesonInterpreterObject):
-    def __init__(self, start: int, stop: int, step: int, *, subproject: T.Optional[str] = None) -> None:
+    def __init__(self, start: int, stop: int, step: int, subproject: str) -> None:
```

Comment:
`subproject` should stay kwarg only (to be consistent with the other calls) but be non-optional

Reply:
right, done.